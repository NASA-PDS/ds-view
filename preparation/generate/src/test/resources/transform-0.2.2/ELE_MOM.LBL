PDS_VERSION_ID                = PDS3                                          
RECORD_TYPE                   = FIXED_LENGTH                                  
RECORD_BYTES                  = 46                                            
FILE_RECORDS                  = 2278                                          
^TABLE                        = "ELE_MOM.TAB"                                 
DATA_SET_ID                   = "VG2-J-PLS-5-SUMM-ELE-MOM-96.0SEC-V1.0"       
SPACECRAFT_NAME               = "VOYAGER 2"                                   
INSTRUMENT_NAME               = "PLASMA SCIENCE EXPERIMENT"                   
TARGET_NAME                   = JUPITER                                       
START_TIME                    = 1979-07-06T00:00:42.687Z                      
STOP_TIME                     = 1979-07-09T23:59:06.436Z                      
MISSION_PHASE_NAME            = "VOYAGER 2 JUPITER ENCOUNTER"                 
PRODUCT_ID                    = "ELE_MOM.TAB"                                 
PRODUCT_TYPE                  = "DATA"                                        
PRODUCT_CREATION_TIME         = "UNK"                                         
SPACECRAFT_CLOCK_START_COUNT  = "2/20541:22:726"                              
SPACECRAFT_CLOCK_STOP_COUNT   = "2/20661:20:726"                              
DESCRIPTION                   = "                                             
  Electron density and moment temperature from the Plasma experiment on       
  Voyager 2 from the Jupiter encounter.  The data set contains 96 second      
  averages of the electron moment data."                                      
                                                                              
OBJECT                        = TABLE                                         
  INTERCHANGE_FORMAT          = ASCII                                         
  ROWS                        = 2278                                          
  COLUMNS                     = 3                                             
  ROW_BYTES                   = 46                                            
  SAMPLING_PARAMETER_NAME     = TIME                                          
  SAMPLING_PARAMETER_UNIT     = SECOND                                        
  SAMPLING_PARAMETER_INTERVAL = 96.0                                          
  OBJECT                      = COLUMN                                        
    NAME                      = "TIME"                                        
    COLUMN_NUMBER             = 1                                             
    UNIT                      = "N/A"                                         
    DATA_TYPE                 = TIME                                          
    START_BYTE                = 1                                             
    BYTES                     = 24                                            
    DESCRIPTION               = "                                             
      Time column. This field contains time in PDS format                     
      yyyy-mm-ddThh:mm:ss.sssZ. The individual elements of the                
      time field can be read using the format                                 
      (i4,4(1x,i2),1x,f6.3) yr, mon, day, hr, min, sec."                      
  END_OBJECT                  = COLUMN                                        
  OBJECT                      = COLUMN                                        
    NAME                      = "ELE_DEN"                                     
    COLUMN_NUMBER             = 2                                             
    UNIT                      = "COUNT/CM**3"                                 
    DATA_TYPE                 = ASCII_REAL                                    
    START_BYTE                = 25                                            
    BYTES                     = 10                                            
    MISSING_CONSTANT          = -9.99e+10                                     
    DESCRIPTION               = "                                             
      Column contains total electron moment density in counts/cm^3."          
  END_OBJECT                  = COLUMN                                        
  OBJECT                      = COLUMN                                        
    NAME                      = "ELE_TEMP"                                    
    COLUMN_NUMBER             = 3                                             
    UNIT                      = "ELECTRONVOLT"                                
    DATA_TYPE                 = ASCII_REAL                                    
    START_BYTE                = 35                                            
    BYTES                     = 10                                            
    MISSING_CONSTANT          = -9.99e+10                                     
    DESCRIPTION               = "                                             
      Column contains total electron moment temperature in units of           
      electron volts."                                                        
  END_OBJECT                  = COLUMN                                        
                                                                              
END_OBJECT                    = TABLE                                         
END                                                                           
