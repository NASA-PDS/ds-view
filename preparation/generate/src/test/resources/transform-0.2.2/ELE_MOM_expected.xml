<?xml version="1.0" encoding="UTF-8"?>
<Product_Observational xmlns="http://pds.nasa.gov/pds4/pds/v1" xmlns:pds="http://pds.nasa.gov/pds4/pds/v1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <Identification_Area>
        <logical_identifier>urn:nasa:pds:data:vg2-j-pls-5-summ-ele-mom-96.0sec-v1.0:ele_mom.tab</logical_identifier>
        <version_id>1.0</version_id>
        <title>vg2-j-pls-5-summ-ele-mom-96.0sec-v1.0 ele_mom.tab</title>
        <information_model_version>1.2.0.1</information_model_version>
        <product_class>Product_Observational</product_class>
        <Modification_History>
            <Modification_Detail>
                <modification_date>2015-08-10</modification_date>
                <version_id>1.0</version_id>
                <description>Auto generated PDS4 product label created by the Transform Tool.</description>
            </Modification_Detail>
        </Modification_History>
    </Identification_Area>
    <Observation_Area>
        <Time_Coordinates>
            <start_date_time>1979-07-06T00:00:42.687Z</start_date_time>
            <stop_date_time>1979-07-09T23:59:06.436Z</stop_date_time>
        </Time_Coordinates>
        <Investigation_Area>
            <name>voyager 2 jupiter encounter</name>
            <type>Mission</type>
            <Internal_Reference>
                <lid_reference>urn:nasa:pds:investigation.voyager.2.jupiter.encounter</lid_reference>
                <reference_type>data_to_investigation</reference_type>
            </Internal_Reference>
        </Investigation_Area>
        <Observing_System>
            <Observing_System_Component>
                <name>plasma science experiment</name>
                <type>Instrument</type>
            </Observing_System_Component>
        </Observing_System>
        <Target_Identification>
            <name>jupiter</name>
            <type>Planet</type>
        </Target_Identification>
    </Observation_Area>
    <File_Area_Observational>
        <File>
            <file_name>ele_mom.tab</file_name>
        </File>
        <Table_Character>
            <offset unit="byte">0</offset>
            <records>2278</records>
            <record_delimiter>Carriage-Return Line-Feed</record_delimiter>
            <Record_Character>
                <fields>3</fields>
                <groups>0</groups>
                <record_length unit="byte">46</record_length>
                <Field_Character>
                    <name>time</name>
                    <field_location unit="byte">1</field_location>
                    <data_type>ASCII_Date_Time</data_type>
                    <field_length unit="byte">24</field_length>
                </Field_Character>
                <Field_Character>
                    <name>ele_den</name>
                    <field_location unit="byte">25</field_location>
                    <data_type>ASCII_Real</data_type>
                    <field_length unit="byte">10</field_length>
                </Field_Character>
                <Field_Character>
                    <name>ele_temp</name>
                    <field_location unit="byte">35</field_location>
                    <data_type>ASCII_Real</data_type>
                    <field_length unit="byte">10</field_length>
                </Field_Character>
            </Record_Character>
        </Table_Character>
    </File_Area_Observational>
</Product_Observational>
